data-share/data-share-service/src/main/java/io/mosip/datashare/util/PolicyUtil.java

```diff
@@ -155,6 +164,17 @@ public DataShareDto getStaticDataSharePolicy(String policyId, String subscriberI
 		}
 	}
 
+	private void validateTransactionAllowed(String transactionsAllowed) {
+		try {
+				int transactions = Integer.parseInt(transactionsAllowed);
+				if(transactions < DataShareServiceImpl.UNLIMITED_TRANSACTION_ALLOWED)

```

Comment:
what happens when the transactionsAllowed is given as 0 in the request ?

Reply:
Added the validation for **usageCountForStandaloneMode** as 0. Now throwing error if it is passed as 0.